backend/curriculum_tracking/models.py

```diff
@@ -1383,3 +1384,24 @@ def add_collaborator(self, user, add_as_project_reviewer):
                 if self.recruit_project:
                     self.recruit_project.reviewer_users.add(user)
         self.save()
+
+    def get_users_that_reviewed_since_last_review_request(self):
```

Comment:
Why is this code in this PR? 

It looks like you are using one branch for 2 features. Please remembert that we are using feature branching. 
I think you'll probably need to do some work to fix your branches. 

Look at the commit network here: https://github.com/Umuzi-org/Tilde/network. The head of one of your branches is stapled onto the tail of another one. That's really not good. 

Those should be 2 seperate branches. Each branch sould come out of develop.

If you need help understanding feature branching and I am not avaiolable I know Ryan understands this stuff well. And I'm sure Vuyi can help too.